rustdoc-json/src/builder.rs

```diff
@@ -33,11 +60,17 @@ pub fn run_cargo_rustdoc(options: Builder) -> Result<PathBuf, BuildError> {
         if manifest.package.is_none() && manifest.workspace.is_some() {
             Err(BuildError::VirtualManifest(options.manifest_path))
         } else {
-            Err(BuildError::General(String::from("See above")))
+            Err(BuildError::CrateBuildError)
```

Comment:
Let's give a hint for future explorers of the code base:

```suggestion
            // In this case the user will see the exact error on stdout or stderr
            Err(BuildError::CrateBuildError)
```